examples/chat.rs

```diff
@@ -205,17 +205,14 @@ fn handle_qp2p_events(
 
 fn print_ourinfo(qp2p: &mut QuicP2p) {
     let ourinfo: Peer = match qp2p.our_connection_info() {
-        Ok(ourinfo) => ourinfo.into(),
+        Ok(addr) => Peer::Node(addr),
         Err(e) => {
             println!("Error getting ourinfo: {}", e);
             return;
         }
     };
 
-    println!(
-        "Our info:\n\n{}\n",
-        serde_json::to_string(&ourinfo).unwrap()
-    );
+    println!("Our info: {}", ourinfo);
```

Comment:
This should to be
```suggestion
    println!("Our info: {}", ourinfo.peer_addr());
```
and the decoding part should be changed accordingly - currently, it expects a JSON-encoded `Peer` struct and the example doesn't work.

Optionally, this can also be encoded with `serde_json::to_string(...)` to simplify decoding.

Reply:
Thanks, fixed now.